tests/pkg_keccak/Makefile

```diff
@@ -0,0 +1 @@
+include $(RIOTBASE)/Makefile.base
```

Comment:
This makefile seems somewhat lacking...

Reply:
You're right, not sure what happened here. I'll fix it later.